prooph/event-store-bus-bridge/3.1/config/packages/prooph_event_store_bus_bridge.yaml

```diff
@@ -0,0 +1,16 @@
+services:
+    _defaults:
+        autowire: true
+
+    Prooph\EventStoreBusBridge\EventPublisher:
+        tags:
+            - { name: 'prooph_event_store.app.plugin' }
+
+    # Uncomment these lines if your event store support transaction
```

Comment:
The default event store `Prooph\EventStore\Pdo\MySqlEventStore` doesn't implement `Prooph\EventStore\TransactionalEventStore`, is it ok for you to have this comment block then @UFOMelkor ?